composites/Plugin/Shared/components/Collapsible.js

```diff
@@ -11,21 +11,35 @@ const StyledContainer = styled.div`
 	background-color: ${ colors.$color_white };
 `;
 
-const StyledContent = styled.div`
-	padding: 0 15px;
-`;
-
 const StyledIconsButton = styled( IconsButton )`
 	width: 100%;
 	background-color: ${ colors.$color_white };
-	padding: 15px;
+	padding: ${ props => props.headingPadding };
```

Comment:
I think we should refrain from creating specific props for styling, can't we use a `styled-components` way to make sure the `StyledIconsButton` has the correct padding for the components that need different paddings.